### ISSUE TITLE
Playerfactions mod for easier area management

### ISSUE THREAD

**Issue by atoms118** [pandorabox-io/pandorabox.io#336]
Currently, in order to control who can access an area you have to manually add them to each area individually.

This works if you have a small number of shared areas, or aren't sharing areas with very many people, but becomes extremely tedious if you have many areas or want to share them with a large number of people.

The `playerfactions` mod solves this issue, by allowing players to create and join factions. Factions are password-protected, so unless you post the password for your faction publicly there won't be issues with new players joining and then never playing again.

In addition to this, `playerfactions` has integration with the `areas` mod. This was just merged into upstream `areas`, and adds `/area_faction_open [ID]`, which behaves the same as `/area_open [ID]` except it only opens/closes the area to your faction as opposed to the entire world.

TL;DR `playerfactions` is a tool for managing access to areas for large groups of players.

The Git repo for playerfactions is [here](https://git.leagueh.xyz/aldenp/playerfactions), and the areas mod simply has to be updated as support is already in upstream.

Thanks for your time.

**Comment by thomasrudin** [comment]
Sounds good, will check when i'm back from vacation... :+1:

**Comment by thomasrudin** [comment]
* areas updated: https://github.com/pandorabox-io/pandorabox-mods/pull/101
* and `playerfactions` added (https://github.com/pandorabox-io/pandorabox-mods/commit/4b1bb136f2c46829bd2a0d2e0565e2947fbed602)

I haven't tested them though, hope that works :crossed_fingers:
